hw/bsp/imxrt/family.c

```diff
@@ -169,6 +190,19 @@ void USB_OTG2_IRQHandler(void)
   #endif
 }
 
+void reboot_to_application(void)
```

Comment:
reboot_to_application() / reboot_to_bootloader() aren't used by tinyusb example, could we dropped it ?